packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/android/initializeflow.md

```diff
@@ -0,0 +1,54 @@
+
+Initialize AuthFoundationBootstrap in your `Application` sublcass. This code shows loading the values from a property file in your project.
```

Comment:
Delete this. It's repeated further down.

Reply:
I slightly modified things as there are two things in this para. The first sentence is definitely a repeat. The second is trying to say that the code example is using a property file. I changed the first para to be one sentence:

> The following example loads the configuration values for the flow from a property file in your project.

It's in the next commit.